include/mbedtls/ctr_drbg.h

```diff
@@ -56,9 +79,19 @@
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16 /**< The block size used by the cipher. */
 
 #if defined(MBEDTLS_CTR_DRBG_USE_128_BIT_KEY)
-#define MBEDTLS_CTR_DRBG_KEYSIZE            16 /**< The key size used by the cipher (compile-time choice: 128 bits). */
+#define MBEDTLS_CTR_DRBG_KEYSIZE            16
+/**< The key size in bytes used by the cipher.
+ *
+ * Compile-time choice: 16 bytes (128 bits)
+ * because #MBEDTLS_CTR_DRBG_USE_128_BIT_KEY is enabled.
```

Comment:
What's the difference between `#MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` and `\c MBEDTLS_CTR_DRBG_USE_128_BIT_KEY`? Since I can see that both of these versions are used.

Reply:
`#MACRO` typesets `MACRO` and links to the definition of `MACRO` if `MACRO` is defined, but typesets `#MACRO` if `MACRO` is undefined, so it isn't good for preprocessor symbols that may or may not be defined. Hence the use of `\c MACRO` for symbols that may or may not be defined.

Reply:
Shouldn't it be `\c MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` everywhere then? Or will doxygen ignore blocks that won't get compiled?

Reply:
It can be `#MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` here because this comment is surrounded by `#ifdef MBEDTLS_CTR_DRBG_USE_128_BIT_KEY`.